libbeat/paths/paths.go

```diff
@@ -142,9 +152,10 @@ func (paths *Path) Resolve(fileType FileType, path string) string {
 		return filepath.Join(paths.Config, path)
 	case Data:
 		return filepath.Join(paths.Data, path)
+	case Logs:
+		return filepath.Join(paths.Logs, path)
 	default:
-		logp.WTF("Unknown file type: %s", fileType)
-		return ""
+		panic(fmt.Sprintf("Unknown file type: %s", fileType))
```

Comment:
That would mean the function would have to return an error, which would complicate usage for little benefit. Note that because FileType is a custom type, the caller won't be able to pass an invalid value here (compile time error). 

The only case where the panic would hit is if someone adds a new entry to the "enum"in this file but forgets to add it here. A panic is fine in that case, IMHO, because it's clearly a development bug that should be discovered before committing.